avro-converter/src/test/java/io/confluent/connect/avro/AdditionalAvroDataTest.java

```diff
@@ -44,7 +46,8 @@ public void testDocumentationPreservedSchema() throws IOException
     {
         Schema avroSchema = new Parser().parse(new File("src/test/avro/DocTestRecord.avsc"));
 
-        org.apache.kafka.connect.data.Schema connectSchema = avroData.toConnectSchema(avroSchema);
+        org.apache.kafka.connect.data.Schema connectSchema = avroData.toConnectSchema(avroSchema,
```

Comment:
These tests wouldn't need to add these extra params if we maintain the public interface for `toConnectSchema`.